nft/nfts.yaml

```diff
@@ -60,6 +62,1097 @@ paths:
+                    description: String - The page key to use to fetch the next page of results. Returns null if there are no more results.
+              examples:
+                nftSales_response:
+                  $ref: '#/components/schemas/nftSales_response'
+      operationId: getNFTSales
+  '/v3/{apiKey}/getContractsForOwner':
+    get:
+      summary: getContractsForOwner
+      description: Gets all NFT contracts held by an owner address.
+      tags: ['Ownership & Token Gating']
+      parameters:
+        - $ref: '#/components/schemas/apiKey'
+        - $ref: '#/components/schemas/owner'
+        - $ref: '#/components/schemas/pageKey'
+        - $ref: '#/components/schemas/pageSize'
+        - $ref: '#/components/schemas/withMetadata'
+        - $ref: '#/components/schemas/includeFilters'
+        - $ref: '#/components/schemas/excludeFilters'
+        - $ref: '#/components/schemas/orderBy'
+        - $ref: '#/components/schemas/spamConfidenceLevel'
+      x-readme:
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  contracts:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/ownedContractv3'
+                  pageKey:
+                    $ref: '#/components/schemas/pageKey'
+                  totalCount:
+                    type: string
+                    description: 'String - Total number of NFT contracts held by the given address.'
+              examples:
+                withoutMetadata:
+                  $ref: '#/components/schemas/withoutMetadataGetContractsForOwner_response'
+                withMetadata:
+                  $ref: '#/components/schemas/withMetadataGetContractsForOwner_response'
+      operationId: getContractsForOwner
+  '/v3/{apiKey}/reportSpam':
+    get:
+      summary: reportSpam
+      description: Report a particular address to our APIs if you think it is spam
+      tags: ['Spam Detection']
+      parameters:
+        - $ref: '#/components/schemas/apiKey'
+        - $ref: '#/components/schemas/address'
+          required: true
+      x-readme:
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: string
+                description: 'String - "Address was successfully reported as spam" if calling the API was successful. '
+      operationId: reportSpam
+## End of v3 methods
+##
+## Below are all the v2 methods
+##
+  '/v2/{apiKey}/getNFTs':
+    get:
+      summary: getNFTs
+      description: Gets all NFTs currently owned by a given address.
+      tags: ['v2']
```

Comment:
@ankg can we change the tag to "NFT API V2 Methods" to make it more descriptive / clear

Reply:
done